bci_tester/data.py

```diff
@@ -1137,11 +1138,19 @@ def create_BCI(
 
 
 if __name__ == "__main__":
-    import json
+    from typing import Set
```

Comment:
Maybe it's just my tired brain, but why are those changes needed for this PR? :thinking:

Reply:
This is the actual code change that checks whether all markers are present in `pyproject.toml`. Before that, it would print the container base urls, but this code path hasn't been used in ages